src/Classes/ServiceAPI/MyRadio_Selector.php

```diff
@@ -196,11 +196,12 @@ public static function setStudio($studio)
             throw new MyRadioException('Selector Locked');
         } elseif ($response === 'ACK') {
             return [
+            'ready' => self::remoteStreams()['ready'],
```

Comment:
Changed it to just true, since hopefully that makes it slightly less broken if something weird happens to remoteStreams().